### DISCUSSION TITLE
Flask not populating

### DISCUSSION THREAD

**Discussion by JBmartin1991** [kyleskom/NBA-Machine-Learning-Sports-Betting#114] [Q&A]
So, just for clarity's sake, while I have decent experience in coding I am brand new to python, so the odds that I'm missing something obvious are pretty high. But, as the title states, after I create the dataset i want and train the models, when I boot up flask there is nothing in the fields. It loads the webpage fine but it just shows the base info (title, headers) with no data. I feel like I'm missing something obvious about where the data populates and is pulled from but I can't find it if so.

**Comment by nkgilley** [comment]
There were no NBA games yesterday. Try again today.

**Comment by JBmartin1991** [comment]
Okay it worked today, thanks! Follow up question though, is there anyway to have it look at games in a future date if the sportsbook has odds already? Like was there somewhere in the code that I could set a it a day or so in the future and look at those games?
